### ISSUE TITLE
Can you make something like this that doesn't depend on Iron Router?

### ISSUE THREAD

**Issue by trusktr** [MiroHibler/meteor-preloader#17]
I'm using [flow-router](https://github.com/meteorhacks/flow-router). It'd be nice to have manual control over the loading so I can write some code in the `action` of a flow-router route to load stuff.

**Comment by MiroHibler** [comment]
From the quick look at flow-router it looks nice (I'm not using it yet) but it would probably require some time to adopt preloader to it and unfortunately I currently don't have that time... :(

But you're very welcome to jump on it ;)

**Comment by emmanuelbuah** [comment]
I agree. Is there any reason why it has to depend on a router? It would be great if it can be configured in and of itself without a router.

**Comment by markudevelop** [comment]
+1

**Comment by tcastelli** [comment]
Not sure if this will fit, but there's a package for router abstraction (compatible with iron router and flowrouter)
https://github.com/nicolaslopezj/meteor-router-layer

**Comment by MiroHibler** [comment]
Thanks! I'm actually starting to delve into [flow-router](https://github.com/meteorhacks/flow-router) and will try to work it out somehow ;)

**Comment by avalanche1** [comment]
+1
